repos/spack_repo/builtin/packages/seacas/package.py

```diff
@@ -263,6 +263,7 @@ class Seacas(CMakePackage):
     # Always depends on netcdf-c
     depends_on("netcdf-c@4.8.0:+mpi+parallel-netcdf", when="+mpi")
     depends_on("netcdf-c@4.8.0:~mpi", when="~mpi")
+    depends_on("netcdf-c@:4.9.2", when="@:2024-08-15")
```

Comment:
This isn't needed.  We have tested current seacas using netCDF-4.9.3 and it works correctly.

Reply:
The change to address the change in netCDF's use of `_FillValue` vs `NC_FillValue` should be in  seacas@2025-02-27 and later.

> @gsjaardema The issue that this was intended to address is reproduced in https://gitlab.spack.io/spack/spack-packages/-/jobs/17663079:
> 
> ```
>   >> 2719    packages/seacas/libraries/exodus/src/ex_put_prop.c:246:54: error: '_FillValue' undeclared (first use in this function); did you mean 'NC_FillValue'?
>      2720      246 |     if ((status = nc_put_att_longlong(exoid, propid, _FillValue, int_type, 1, vals)) != NC_NOERR) {
>      2721          |                                                      ^~~~~~~~~~
>      2722          |                                                      NC_FillValue
> ```

Reply:
Just got off long vacation and my reading comprehension is obviously not yet focusing on details yet...  Sorry for the confusing replies above...

Reply:
Ok, I reverted this to include the constraint (d7a3e8983c1), where `2024-08-15` is the last release before `2025-02-27`.